ui/src/onboarding/reducers/dataLoaders.ts

```diff
@@ -34,6 +34,7 @@ export const INITIAL_STATE: DataLoadersState = {
   precision: WritePrecision.Ns,
   telegrafConfigID: null,
   pluginBundles: [],
+  scraper: {interval: '', bucket: '', urls: []},
```

Comment:
I wonder if DataLoadersState should move to a more nested state in the future ?
```
{
streaming:...,
line protocol:..., 
scraping:..,
}
```

Reply:
i definitely agree with this. after alpha, i believe the wizards will be separating out a lot and that feels like a nice time to make this change